### DISCUSSION TITLE
ALERT!!! Script/Phonzy.A!ml Virus detected in the Codeproject.ai 2.9.3 installer

### DISCUSSION THREAD

**Discussion by alliasx** [codeproject/CodeProject.AI-Server#212] [General]
Trying to determine how this virus got embedded. Downloaded from "[CodeProject.AI Server - CodeProject.AI Server v2.9.2](https://codeproject.github.io/)" prerelease. Please run your virus scanners.
CodeProject.AI-Server_2.9.3.1_win_x64.zip

**Comment by ChrisMaunder** [comment]
It's a false positive. Try using the 2.9.4 installer
